sda-commons-web-autoconfigure/src/test/java/org/sdase/commons/spring/boot/web/security/test/SecurityTestApp.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022- SDA SE Open Industry Solutions (https://www.sda.se)
+ *
+ * Use of this source code is governed by an MIT-style
+ * license that can be found in the LICENSE file or at
+ * https://opensource.org/licenses/MIT.
+ */
+package org.sdase.commons.spring.boot.web.security.test;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.Context;
+import org.sdase.commons.spring.boot.web.error.ApiException;
+import org.sdase.commons.spring.boot.web.jackson.EnableSdaRestGuide;
+import org.sdase.commons.spring.boot.web.security.EnableSdaWebSecurity;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+@SpringBootApplication
+@RestController
+@EnableSdaRestGuide
+@EnableSdaWebSecurity
+public class SecurityTestApp {
```

Comment:
I tried to compare the suggested annotations because I expected only one SDA annotation `@EnableSdaPlatform`. 

I went through the documentation but I could not find a nice howto that tells me how I write a service using Spring Boot Commons. Is [this](https://github.com/SDA-SE/sda-spring-boot-commons/blob/master/docs/web/index.md) the right place to search for the documentation?

Reply:
We should not need any of these annotations due to auto configuration. I'll take a look how to use auto configuration for the security settings.

Yes, that is the right place for documentation. Maybe we should also add a quick start page, where we document the bare minimum of a `build.gradle`, the application class and the `application.properties`. But imho that would be another PR.

Reply:
Maybe this information help us creating our own auto config? 
https://www.baeldung.com/spring-boot-custom-auto-configuration#creating-a-custom-auto-configuration

Reply:
Imho that article is already outdated. `spring.factories` [has been replaced](https://github.com/spring-projects/spring-boot/issues/29872) by [the config](https://github.com/SDA-SE/sda-spring-boot-commons/blob/master/sda-commons-web-autoconfigure/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports) we are using..